newIDE/app/src/BehaviorsEditor/EmptyBehaviorsPlaceholder.js

```diff
@@ -4,25 +4,41 @@ import * as React from 'react';
 import HelpButton from '../UI/HelpButton';
 import Text from '../UI/Text';
 import { EmptyPlaceholder } from '../UI/EmptyPlaceholder';
+import Add from '@material-ui/icons/Add';
+import { Column, Line } from '../UI/Grid';
+import RaisedButton from '../UI/RaisedButton';
 
-const EmptyBehaviorsPlaceholder = () => (
+type Props = {
+  openNewBehaviorDialog: () => void,
+};
+
+const EmptyBehaviorsPlaceholder = (props: Props) => (
   <EmptyPlaceholder
-    renderButtons={() => <HelpButton helpPagePath="/behaviors" />}
+    renderButtons={() => (
+      <HelpButton
+        label={<Trans>Read the doc</Trans>}
+        helpPagePath="/behaviors"
+      />
+    )}
+    buttonJustification="center"
+    outlined={false}
   >
-    <Text>
-      <Trans>There are no behaviors here.</Trans>
-    </Text>
-    <Text>
-      <Trans>
-        Behaviors are predefined actions that are assigned to objects. Behaviors
-        can have no or multiple parameters.
-      </Trans>
+    <Text size="title" align="center">
+      <Trans>Add your first behavior</Trans>
     </Text>
-    <Text>
-      <Trans>
-        Add your first behavior using the button "Add a behavior to the object".
-      </Trans>
+    <Text align="center">
+      <Trans>Behaviors add features to objects in a matter of clicks.</Trans>
     </Text>
+    <Line justifyContent="center" expand>
+      <RaisedButton
+        key="add-behavior-line"
+        label={<Trans>Add a behavior</Trans>}
+        primary
+        onClick={() => props.openNewBehaviorDialog()}
```

Comment:
this can probably be just `onClick={props.openNewBehaviorDialog}`

Reply:
Ok, I made the change.

When a behavior is added, the view is the same as before. The "add" button is on bottom right.